package.json

```diff
@@ -0,0 +1,154 @@
+{
+  "name": "@ipfs/public-gateway-checker",
+  "version": "1.0.0",
+  "description": "Checks which public gateways are online or not",
+  "author": "Marcin Rataj <lidel@lidel.org>",
+  "license": "Apache-2.0 OR MIT",
+  "homepage": "https://github.com/ipfs/public-gateway-checker#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ipfs/public-gateway-checker.git"
+  },
+  "bugs": {
+    "url": "https://github.com/ipfs/public-gateway-checker/issues"
+  },
+  "engines": {
+    "node": ">=16.0.0",
+    "npm": ">=7.0.0"
+  },
+  "main": "src/index.ts",
+  "files": [
+    "src",
+    "dist"
+  ],
+  "eslintConfig": {
+    "extends": "ipfs"
+  },
+  "release": {
+    "branches": [
+      "master"
+    ],
+    "plugins": [
+      [
+        "@semantic-release/commit-analyzer",
+        {
+          "preset": "conventionalcommits",
+          "releaseRules": [
+            {
+              "breaking": true,
+              "release": "major"
+            },
+            {
+              "revert": true,
+              "release": "patch"
+            },
+            {
+              "type": "feat",
+              "release": "minor"
+            },
+            {
+              "type": "fix",
+              "release": "patch"
+            },
+            {
+              "type": "chore",
+              "release": "patch"
+            },
+            {
+              "type": "docs",
+              "release": "patch"
+            },
+            {
+              "type": "test",
+              "release": "patch"
+            },
+            {
+              "scope": "no-release",
+              "release": false
+            }
+          ]
+        }
+      ],
+      [
+        "@semantic-release/release-notes-generator",
+        {
+          "preset": "conventionalcommits",
+          "presetConfig": {
+            "types": [
+              {
+                "type": "feat",
+                "section": "Features"
+              },
+              {
+                "type": "fix",
+                "section": "Bug Fixes"
+              },
+              {
+                "type": "chore",
+                "section": "Trivial Changes"
+              },
+              {
+                "type": "docs",
+                "section": "Trivial Changes"
+              },
+              {
+                "type": "test",
+                "section": "Tests"
+              }
+            ]
+          }
+        }
+      ],
+      "@semantic-release/changelog",
+      "@semantic-release/npm",
+      "@semantic-release/github",
+      "@semantic-release/git"
+    ]
+  },
+  "scripts": {
+    "start": "npx serve -l 3000 dist",
+    "copy-assets": "cp-cli src/index.html dist/index.html && cp-cli src/styles.css dist/styles.css",
+    "lint": "aegir lint && aegir ts -p check",
+    "release": "aegir release",
+    "publish-to-ipfs": "jsipfs add -wrQ dist > lastpubver",
```

Comment:
This project uses GH Pages for now, this is not used, please remove `lastpubver` too
```suggestion
```